docs/Vertical-Rhythm.md

```diff
@@ -0,0 +1,47 @@
+# Vertical Rhythm
+
+_Vertical Rhythm is an important concept in web design. It has the ability to bring a design together and make different elements feel consistent on the same page_
+
+## Challenge
+
+When rendering a page using shared components it is up to the renderer to decide on the spacing between the components, AKA _Vertical Rhythm_. This means the renderer will need to know the expected spacing between different components. 
+
+## Proposed Solution
+
+A component can only define the spacing below itself rather than both above and below. This would allow for the desired flexibility of any component followed by any other component.
```

Comment:
```suggestion
A component can only define the spacing below itself rather than both above and below. This would allow for the desired flexibility of any component followed by any other component.

That said, we need to confirm with UX the edge cases where a component has a large padding-bottom and is followed by a component which defines padding-top for additional spacing. 

**Example case**
A `<h1>` followed by a `<h2>` where:
- The `<h1>` has padding-bottom of 40px
- The `<h2>` expects a spacing of 32px above it
- The `<h2>` assumes the component above has a padding-bottom of 24px (the default spacing used by majority of components)

Based on the assumption the `<h2>` has a top padding of 8px (default bottom 24px + additional top padding 8px = desired 32px). However, rendered spacing is actually 48px rather than 32px due to difference between the _default padding-bottom_ and the _`<h1>`'s padding-bottom_.
```